requirements/requirements.test.txt

```diff
@@ -4,25 +4,25 @@
 #
 #    pip-compile --output-file requirements.test.txt requirements.test.in
 #
-certifi==2018.1.18        # via requests
-chardet==3.0.4            # via requests
+# certifi==2018.1.18        # via requests
+# chardet==3.0.4            # via requests
 coverage==4.0.3
 django-debug-toolbar==1.9.1
-django==2.0.4             # via django-debug-toolbar
+# django==2.0.4             # via django-debug-toolbar
 flake8==3.5.0
 httmock==1.2.6
-idna==2.6                 # via requests
+# idna==2.6                 # via requests
 mccabe==0.6.1             # via flake8
 pluggy==0.6.0             # via tox
 py==1.5.3                 # via tox
 pycodestyle==2.3.1        # via flake8
 pyflakes==1.6.0           # via flake8
 python-coveralls==2.9.1
-pytz==2018.3              # via django
+# pytz==2018.3              # via django
 pyyaml==3.12              # via python-coveralls
-requests==2.18.4          # via httmock, python-coveralls
-six==1.11.0               # via python-coveralls, tox
+# requests==2.18.4          # via httmock, python-coveralls
+# six==1.11.0               # via python-coveralls, tox
 sqlparse==0.2.4           # via django-debug-toolbar
 tox==3.0.0
-urllib3==1.22             # via requests
+# urllib3==1.22             # via requests
```

Comment:
Just so I understand, these are commented out because they're also in requests.txt? This is fine, but I worry about hanging a generated file like this. It looks like there's an alternative way of doing this: https://github.com/jazzband/pip-tools/issues/398.

Anyway, I actually don't think this should be a blocker, but this is just something to keep in mind. Definitely a nice to have.

Reply:
I updated it so that the dev requirements include all of the normal requirements. Not sure if I like that better than having them seperate in both the `in` and `txt` files but it works.